src/pages/patientView/PatientViewPage.tsx

```diff
@@ -112,130 +158,62 @@ export default class PatientViewPage extends React.Component<
-        reaction(
-            () => [
-                props.routing.query,
-                props.routing.location.hash,
-                props.routing.location.pathname,
-            ],
-            ([query, hash, pathname]) => {
-                // we don't want to update patient if we aren't on a patient page route
-                if (!pathname.includes('/' + PagePath.Patient)) {
-                    return;
-                }
-
-                const validationResult = validateParametersPatientView(query);
-
-                if (validationResult.isValid) {
-                    this.patientViewPageStore.urlValidationError = null;
-
-                    if ('studyId' in query) {
-                        this.patientViewPageStore.studyId = query.studyId;
-                    }
-                    if ('caseId' in query) {
-                        this.patientViewPageStore.setPatientId(
-                            query.caseId as string
-                        );
-                    } else if ('sampleId' in query) {
-                        this.patientViewPageStore.setSampleId(
-                            query.sampleId as string
-                        );
-                    }
-
-                    // if there is a navCaseId list in url
-                    const navCaseIdMatch = hash.match(/navCaseIds=([^&]*)/);
-                    if (navCaseIdMatch && navCaseIdMatch.length > 1) {
-                        this.patientViewPageStore.patientIdsInCohort = parseCohortIds(
-                            navCaseIdMatch[1]
-                        );
-                    }
-                } else {
-                    this.patientViewPageStore.urlValidationError =
-                        validationResult.message;
-                }
-            },
-            { fireImmediately: true }
-        );
-
-        this.mergeMutationTableOncoKbIcons = this.patientViewPageStore.mergeOncoKbIcons;
-    }
-
-    @computed get mergedMutations() {
-        // remove fusions
-        return this.patientViewPageStore.mergedMutationDataIncludingUncalledFilteredByGene.filter(
-            mutationArray => {
-                return !isFusion(mutationArray[0]);
-            }
-        );
-    }
-
-    @computed get mergedCnas() {
-        return this.patientViewPageStore.mergedDiscreteCNADataFilteredByGene;
-    }
-
-    componentDidMount() {
-        // Load posted data, if it exists
-        const postData = getBrowserWindow().clientPostedData;
-        if (postData && postData.navCaseIds) {
-            this.patientViewPageStore.patientIdsInCohort = buildCohortIdsFromNavCaseIds(
-                postData.navCaseIds
-            );
-            getBrowserWindow().clientPostedData = null;
-        }
-    }
-
-    public get showNewTimeline() {
-        return !getServerConfig().patient_view_use_legacy_timeline;
-    }
-
-    public get showOldTimeline() {
-        return getServerConfig().patient_view_use_legacy_timeline;
```

Comment:
I guess we don't support the legacy timeline anymore, right?

Reply:
correct.  long gone.  probably should kill the property